pkg/actors/actors_test.go

```diff
@@ -14,1240 +14,16 @@ limitations under the License.
 package actors
 
 import (
-	"context"
-	"encoding/json"
-	"fmt"
-	"net/http"
```

Comment:
Was this file meant to be deleted or intentionally left to add tests back in at a later time?

Reply:
The tests in this file have been moved to the relevant places or removed as they are no longer relevant.